src/@next/Tabs/Tabs.tsx

```diff
@@ -109,15 +112,21 @@ export const Tabs = React.forwardRef<HTMLDivElement, TabsProps>(function Tabs(
 
   const renderTabs = tabs.map((tab: TabModel, index: number) => {
     const tabId = tab.id || nextId();
+
+    const tabContent = (
+      <Tab
+        id={tabId}
+        key={`tab-${tabId}-${index}`}
+        content={tab.content}
+        onSelect={() => handleSelectedIndexChanged(index)}
+        selected={index === selectedTabIndex}
+        disabled={tab.disabled}
+      ></Tab>
+    );
+
     return (
       <StyledLi key={`${tabId}-${index}`} className="tab-item">
-        <Tab
-          id={tabId}
-          key={`tab-${tabId}-${index}`}
-          content={tab.content}
-          onSelect={() => handleSelectedIndexChanged(index)}
-          selected={index === selectedTabIndex}
-        ></Tab>
+        {tab.contentWrapper ? tab.contentWrapper(tabContent) : tabContent}
```

Comment:
Reason on why I don't place the wrapper outside StyledLi is, it will break current styling and will add more changes that may be risky.

Having it inside the StyledLi doesn't affect the styling.